packages/calcite-components/src/components/date-picker-day/date-picker-day.scss

```diff
@@ -42,11 +43,11 @@
 @mixin range-part-middle() {
   &:before {
     inset-inline-end: 50%;
-    border-radius: 0;
+    border-radius: var(--calcite-corner-radius);
   }
   &:after {
     inset-inline-start: 50%;
-    border-radius: 0;
+    border-radius: var(--calcite-corner-radius);
```

Comment:
I don't think these `border-raidus` needs to be themable. These should be set to `0` to build range kind of UI. Couldn't think of an use-case where users wants to set it to any value other than `0`.  Thoughts? @macandcheese / @SkyeSeitz

Reply:
Additionally, the hover region will be using square edges as part of enhancement #3455 .

Reply:
Theming `color` and `background-color` might be the priority for now? `border-radius` Seems like it would be difficult to keep the middle dates of the ranges from peaking out of the rounded off edges, especially on a responsive Date Picker that can grow and shrink.